### ISSUE TITLE
Isfet ability issues

### ISSUE THREAD

**Issue by Mmlh1** [cigumo/krvi#402]
- Meteors:
 - [ ] They have a tiny area of 30 (diameter) and hardly ever hit anything. Area 80 would be much better.
 - [ ] They have a 70s cooldown. You can use the hero spell more often than this and that is significantly better.
 - [ ] You cannot even properly choose where to use them since they will activate while moving, so they will activate while walking your hero over to where you want to use them.
 
 - Black Plague:
 - [ ] It keeps hitting enemies while moving through it, so it deals more damage than expected (26 or 27 ticks rather than 20). The description should either be updated to match, or the aura should have exclude_modifier_id to avoid reapplying the same modifier. In the second case the cooldown should be decreased, because 45 seconds without the increased damage is too weak.
 - [ ] Even accounting for the extra hits, the localization assumes 3/5/8 damage per tick rather than 4/6/9, so this needs to be fixed either way.
 
 - Polymorph:
 - [ ] This targets enemies already hit by the Wicked Sisters polymorph, which feels weird.
 
 - Blood Pool:
 - [x] The bubbles fx last a bit longer than the pool fx.

**Comment by pablorealini** [comment]
**Black Plague**
The DOD, like this one, is something you cannot calculate with exactitude and will vary each time depending on the situation. The description states 60, 100, and 160, which is definitely incorrect. It's probable that the game designer incremented the damage but didn't update the description.

My suggestion is to change the description to 110, 165, 250, maybe using phrases like 'up to 110,' etc. These values represent the maximum damage the skill can provide if it lasts 7 seconds (the duration of the aura) and the enemy remains stopped inside the aura, refreshing the 5-second duration. There might be a little more damage:

4 * (1 / 0.25) * 7 = 112 (110 is a better number)
6 * (1 / 0.25) * 7 = 168 (165 is better)
9 * (1 / 0.25) * 7 = 252 (250 is better)

Regarding the exclusion of the modifier ID, it might feel unfair to have an enemy inside the aura unaffected. Additionally, this alteration would change the aura damage the skill currently inflicts to 5 seconds instead of 7 (80, 120, 180).

**Comment by pablorealini** [comment]
**Polymorph**
"I won't fix this because I don't believe it's broken. KR has always functioned this way, allowing an endless number of combinations involving two different entities targeting the same unit.

**Comment by pablorealini** [comment]
**Meteors**
We discussed this with Gerson, and he wants to keep this skill able to be cast while moving. He also didn't want to increase the range of each projectile or the cooldown. I'll talk to him again.

**Comment by Mmlh1** [comment]
**Black Plague**
It moves in the opposite direction to enemies, so it is effectively impossible to keep an enemy in there for the full 7 seconds. It will basically always be 26 or 27 ticks, and it is at least 20 (5 seconds). Hence, giving it exclude_modifier_id will make it fully consistent at 20 ticks, so that the localization will always be correct, and the damage or cooldown can be nudged to compensate for potential damage loss.

**Polymorph**
This one specifically is weird because the Witch frog and Isfet frog use the exact same graphics, so it is confusing to players. I do understand they have different effects and the fact they both target the same thing is not an issue per se, it is purely the graphics being identical.

**Meteors**
Every single player I have talked to about the DLC has complained that the meteors do nothing. They will fail to hit anything whatsoever 90% of the time. Thanks for trying to convince Gerson!

**Comment by pablorealini** [comment]
I agree on black Plague, but it will change the current balance in the game, is a lot easier to keep the balance and change the description than to include a GD to test and rebalance everything to 5 seconds.

**Comment by Mmlh1** [comment]
Fair point.

**Comment by pablorealini** [comment]
Will talk to Gerson about **Black plague** also.
